src/datagrid/components/Table/styles.ts

```diff
@@ -13,18 +16,36 @@ export const DataGrid_Spacer = {
     },
 };
 
-export const DataGrid_MetaWrap = {
-    flexDirection: 'row',
-    borderColor: 'rgb(202, 196, 208)',
-    variants: {
-        // top: { paddingTop: 'spacings.3' },
-        // bottom: { paddingTop: 'spacings.3' },
-    },
-};
-
-export const DataGrid_Meta = {
+export const DataGrid_RowItem = {
```

Comment:
We should break down the styles and have more explicit naming conventions.

Header and Footer are only applicable for the groups, so, we can name the styles something like DataGrid_GroupHeader